packages/integration-sdk-cli/src/neo4j/neo4jGraphStore.ts

```diff
@@ -109,17 +109,19 @@ export class Neo4jGraphStore {
       if (relationship._mapping) {
         //Mapped Relationship
         if (relationship._mapping['skipTargetCreation'] === false) {
+          const targetEntity = relationship._mapping['targetEntity'];
           //Create target entity first
           const tempEntity: Entity = {
-            _class: relationship._mapping['targetEntity']._class,
+            ...targetEntity,
+            _class: targetEntity._class,
```

Comment:
We don't need to worry about sanitizing _class or _type here, since they will be sanitized in the later addEntities() call.